programs/CMakeLists.txt

```diff
@@ -18,6 +18,12 @@ option (ENABLE_CLICKHOUSE_SERVER "Server mode (main mode)" ${ENABLE_CLICKHOUSE_A
 option (ENABLE_CLICKHOUSE_CLIENT "Client mode (interactive tui/shell that connects to the server)"
     ${ENABLE_CLICKHOUSE_ALL})
 
+if (CLICKHOUSE_SPLIT_BINARY)
+    option (ENABLE_CLICKHOUSE_SELF_EXTRACTING "Self-extracting executable" OFF)
+else ()
+    option (ENABLE_CLICKHOUSE_SELF_EXTRACTING "Self-extracting executable" ON)
+endif ()
```

Comment:
```suggestion|
option (ENABLE_CLICKHOUSE_SELF_EXTRACTING "Self-extracting executable" ON)
if (CLICKHOUSE_SPLIT_BINARY)
    set (ENABLE_CLICKHOUSE_SELF_EXTRACTING OFF)
endif ()
```

Reply:
same as above, I guess - let's keep it uniform?

Reply:
agree!